jenkins/run-pytests

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
This file needs to be executable.  The latest test runs failed with, `/usr/bin/bash: ./jenkins/run-pytests: Permission denied`.

Reply:
This file is removed so maybe I can resolve this review